### ISSUE TITLE
Remote desktop can default to 1:1 ratio

### ISSUE THREAD

**Issue by nil-ref** [quasar/Quasar#1139]
### Problem description

I hope that when remote desktop starts, the size of the picbox can be adjusted to match the remote resolution.

### Proposal

int screenWidth = Screen.PrimaryScreen.Bounds.Width;
int screenHeight = Screen.PrimaryScreen.Bounds.Height;
if(screenWidth > d.Resolution.Width && screenHeight > d.Resolution.Height)
    this.ClientSize = new Size(d.Resolution.Width, d.Resolution.Height);

**Comment by Subjoel** [comment]
yes pls, i have the issue when testing that only a part of their screen is shown

**Comment by Subjoel** [comment]
make it a pull request
